02week/blog/css/style.css

```diff
@@ -0,0 +1,192 @@
+
```

Comment:
Nice job.  I like to see those comments.  The only note is to clean up the formatting.